.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,4 +14,5 @@
 
 <!-- Is this a visible change? You probably need to update docs! -->
```

Comment:
```suggestion
<!-- Could this change affect a user's behavior? Ping @withastro/maintainers-docs and let them know! -->
```
@sarah11918 Thoughts on this tweak?

Reply:
You read my mind, @Jutanium! 🙌 

The key idea for me is: 
- will a user (have to) do something differently now? (Docs can decide if that conflicts with current documentation - no need to for devs to worry about that!)

This includes might a user WANT TO do something differently now? e.g. We have a better simpler way because of a new feature or bug fix that we would now recommend instead, even if this isn't a breaking change and the old way still works.

I'm happy with the shortest "trigger" for people that gets them thinking about this when deciding whether or not to ping docs. Especially when thinking about external PRs received, it needs to be "enough" for someone outside the normal process to make that connection. I think this does that!

Reply:
I was going to say, you could go a step further here and make the instructions more clear for the author:

```
<!-- DON'T DELETE THIS SECTION! If no docs added, explain why.-->
<!-- If adding docs (or unsure!) uncomment the next line: -->
<!-- /cc @withastro/maintainers-docs for feedback! -->
```

Or something to that effect!

Reply:
Pushed a commit combining these suggestions — let me know what you all think!